.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Direct_Java11.groovy

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import PhraseTriggeringPostCommitBuilder
+
+
+// This job runs the suite of ValidatesRunner tests against the Dataflow
+// runner.
+PhraseTriggeringPostCommitBuilder.postCommitJob('beam_PostCommit_Java_ValidatesRunner_Direct',
```

Comment:
Right now, this will only run when triggered manually from a PR comment. Should we also add a cron job for this (scheduled eg. once a day)? Or maybe it's better to have this tested post commit?

@aaltay @pabloem and @reuvenlax  what do you think? 

Tldr for you: this job tests if Direct runner's ValidatesRunner suite built with Java8 can be run with Java11, as we discussed. It's purpose is to ensure that new features developed with Java8 will stay compatible when running with Java 11. Thanks to running this suite, we were able to identify and fix a problem with schemas: [BEAM-6705](https://issues.apache.org/jira/browse/BEAM-6705)

Reply:
I think running a post commit would be good. To prevent regressions over time.

Reply:
Agree that we should run this continuously. It being DirectRunner shouldn't take too much time either.

Reply:
I felt the same, just wasn't sure. Thanks for opinions!